src/components/Hero/CountryHero.js

```diff
@@ -20,23 +20,23 @@ const styles = theme => ({
   root: {
     flexGrow: 1,
     marginTop: '32px',
-    [theme.breakpoints.up('md')]: {
+    [theme.breakpoints.up('lg')]: {
```

Comment:
Don't switch `md` to `lg`, this will make `md` look _mobile_ and not _desktop_ (which is what we want).